pkg/api/validation/dynakube/modules.go

```diff
@@ -25,7 +25,7 @@ func isOneAgentModuleDisabled(_ context.Context, v *Validator, dk *dynakube.Dyna
 }
 
 func isActiveGateModuleDisabled(_ context.Context, v *Validator, dk *dynakube.DynaKube) string {
-	if dk.ActiveGate().IsEnabled() && !v.modules.ActiveGate {
+	if dk.ActiveGate().IsEnabled() && !v.modules.ActiveGate && !v.modules.KSPM {
```

Comment:
Bit confusing, but necessary 🥲

Reply:
should we abstract it into a method with a meaningful name to avoid confusion? Or would that overdo it?

Reply:
It's fine as is